controller/project/manager.py

```diff
@@ -34,12 +35,44 @@
 from controller.transfer.manager import check_and_add_running_id
 from controller.upload_task import manager as upload_task_manager
 from controller.gates import gates_service
+from controller.auth import kratos
+from submodules.model.util import sql_alchemy_to_dict
+
+LABELING_TASK_WHITELIST = {
+    "id",
+    "name",
+    "taskTarget",
```

Comment:
camel case, i think this is wrong?
below the same for attribute whitelist and information source whitelist

Reply:
removed unused WHITELISTS